pyquil/tests/test_quil.py

```diff
@@ -1385,12 +1385,10 @@ def test_params_pi_and_precedence():
     exp = str(prog[0].params[0])
     assert _eval_as_np_pi(trivial_pi) == _eval_as_np_pi(exp)
 
-    less_trivial_pi = "3 * theta[0] * 2 / (pi)"
     prog = Program(f"RX({trivial_pi}) 0")
     exp = str(prog[0].params[0])
     assert _eval_as_np_pi(trivial_pi) == _eval_as_np_pi(exp)
```

Comment:
perhaps the line below was actually meant to `RX({less_trivial_pi})` and then `_eval_as_np_pi(less_trivial_pi)` on line 1391?